src/python/crash_analysis/severity_analyzer.py

```diff
@@ -104,8 +104,20 @@ def get_security_severity(crash_type, crash_output, job_name,
 class SeverityAnalyzerSanitizer(object):
   """Generic ASan severity analyzer."""
 
-  def analyze(self, crash_type, _, requires_gestures):
+  def analyze(self, crash_type, crash_output, requires_gestures):
     """Return a security severity based on the ASan crash output."""
+
+    manual_severity_match = re.search(
+        r'fuzzer-security-issue(-critical|-high|-medium|-low|)', crash_output,
+        re.IGNORECASE)
+    if manual_severity_match:
+      manual_severity = manual_severity_match.groups()[0].lower()
+      if manual_severity:
+        # Skip over the leading dash.
```

Comment:
This seems a bit hacky. Why is this needed?

Reply:
This is done to support both `fuzzer-security-bug` (no severity specified, just that this is a security issue) and `fuzzer-security-bug-critical` (severity specified). I could either

1. make the code less condensed by checking for `fuzzer-security-bug-<severity>` first and if not found checking for `fuzzer-security-bug` or
2. just drop the `fuzzer-security-bug` feature so that fuzz target authors need to specify a severity as well.

Reply:
The problem went away per your other comment ;-)